jestconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "collectCoverage": true,
+  "testEnvironment": "node"
```

Comment:
@gigicupp this looks good. I am wondering if the "collectCoverage" key is what generates all these coverage reports and makes every test generate so many files. This is something we should both research because the coverage report files are not necessary to keep in our repo, at least I don't think they are necessary.

Reply:
@masoncpott you are absolutely right on this. I went to the official Jest docs and it says having "true" creating those files. So changed it to false.